<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-F3GJ7JN" failures="1" tests="2" name="Default test(failed)" time="1.104" errors="0" timestamp="2022-05-09T14:02:56 CDT">
  <testcase classname="practice.TestNgRetryFailed" name="test3" time="0.316">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at practice.TestNgRetryFailed.test3(TestNgRetryFailed.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="practice.TestNgRetryFailed" name="test2" time="0.024"/>
  <testcase classname="practice.TestNgRetryFailed" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
